README.md

```diff
@@ -0,0 +1,42 @@
+# Telegram Bot API typings for Flow, TypeScript and Rust
+
+This repository contains [Telegram Bot API](https://core.telegram.org/bots/api) typings for [Flow](https://flow.org/), [TypeScript](https://www.typescriptlang.org/) and [Rust](https://www.rust-lang.org/).
+
+The types are automatically generated for all supported languages from the Telegram Bot API website.
+
+## Flow and TypeScript typings
+
+See [javascript/](javascript/) folder.
+
+## Rust typings
+
+See [rust](rust/) folder.
+
+## Contributing
+
+Source code for the type generation lives under [lib/](lib/) folder.
+
+### Setting up local development environment
+
+To contribute to this project, you will need to have the following tools installed:
+
+* Rust
+* Node v8.x or higher
+* npm v5.7.1 or higher
+
+Once these tools are installed, you can install the required dependencies:
+
+```
+npm install
+cargo install rustfmt --version 0.9.0
```

Comment:
Without the version constraing, `cargo install rustfmt` ended up installing v0.10.0. That ended up saying that `rustfmt` is deprecated and I'd need to pass `--force` flag to get it to work.

I figured you are using v0.9.0 or some earlier version locally, so I put the version constraint here to avoid having to change the `build` npm script to pass `--force` flag to `rustfmt` call.